kura/org.eclipse.kura.core.keystore/src/main/java/org/eclipse/kura/core/keystore/FilesystemKeystoreServiceImpl.java

```diff
@@ -223,17 +230,26 @@ public void deactivate() {
     }
 
     private boolean keystoreExists(String keystorePath) {
-        boolean result = false;
+        return keystorePath != null && new File(keystorePath).exists();
```

Comment:
From the Java documentation:

> public boolean exists​()
> Tests whether the file **or directory** denoted by this abstract pathname exists.

Should we check also for the type of the file?

```suggestion
        return keystorePath != null && new File(keystorePath).isFile();
```

From the docs:
> public boolean isFile​()
> ....
>true if and only if the file denoted by this abstract pathname **exists and is a normal file**; false otherwise